app/api/signup/views.py

```diff
@@ -0,0 +1,16 @@
+from fastapi import APIRouter
+from .schemas import Signup
+
+
+router = APIRouter()
+
+
+@router.post("/", response_model= Signup)
+async def signup_user(first_name: str,last_name: int, email: str, password: str):
```

Comment:
Defining the parameters here makes them query parameters, they need to be in the request body.

See https://fastapi.tiangolo.com/tutorial/body/#declare-it-as-a-parameter

Reply:
Okay. I have done that:  Making parameters in the request body and example and system API routers restored.